virt/virt-2-4-release-notes.adoc

```diff
@@ -147,8 +147,21 @@ $ oc delete pod -n openshift-cnv --selector=kubevirt.io=virt-handler --field-sel
 ----
 <1> Where <source-node-name> is the name of the source node that the VMI is migrating from.
 
-//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
-* If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
+* If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859235[*BZ#1859235*])
+
+** To verify that a virtual machine has been created from a newer version of the common templates, ensure that the `spec.terminationGracePeriodSeconds` value is not `0` by running the following command in the namespace of the virtual machine:
++
+[source,terminal]
+----
+$ oc get vm <virtual-machine-name> -o yaml | grep "terminationGracePeriodSeconds"
+----
+
+** If the virtual machine has a `terminationGracePeriodSeconds` of `0`, patch the virtual machine config with a `spec.terminationGracePeriodSeconds` value of `180` for Linux-based virtual machines, or a value of `3600` for Windows-based virtual machines.
```

Comment:
This step should also be applied to VMs which were created prior to the upgrade.

Reply:
@rnetser Added a note and slightly rewrote so that users with VMs created from common templates prior to upgrade will also pay attention to the workaround